src/main/java/com/google/sps/servlets/SignUpServlet.java

```diff
@@ -25,7 +25,7 @@
 public class SignUpServlet extends HttpServlet {
 
 	// Check if user with specified field exists
-	static private boolean checkIfFieldExists(String field, String value) {
+	static private boolean checkIfFieldExists(String field, String value, Datastore datastore) {
```

Comment:
Does this method need to be static?

Reply:
> Do you need to specifically add the .class files?

I realize now that I don't so i removed that in commit e0f47a5

Reply:
I don't see the loop changes in that commit, only the deleted .class files.

Reply:
Should be in 
d5642bb

Reply:
Looks like in that change you modified the SignInServlet, not the SignOutServlet. Different loops.

Reply:
Sorry it should be in this new commit 38c4a57